sea-orm-macros/src/lib.rs

```diff
@@ -539,6 +539,19 @@ pub fn derive_active_model_behavior(input: TokenStream) -> TokenStream {
 ///         - For `string_value`, value should be passed as string, i.e. `string_value = "A"`
 ///         - For `num_value`, value should be passed as integer, i.e. `num_value = 1` or `num_value = 1i32`
 ///         - Note that only one of it can be specified, and all variants of an enum have to annotate with the same `*_value` macro attribute
+///
+/// # Usage
+///
+/// ```
+/// use sea_orm::{sea_query, EnumIter, DeriveActiveEnum};
```

Comment:
Now we do not need to import Iden, EnumIter nor ActiveEnum, but sea_query is still needed.